neuroio/api/entries/v1.py

```diff
@@ -26,22 +27,16 @@ def list(
         offset: int = 0,
     ) -> Response:
         data = request_query_processing(locals(), ["self"])
-        try:
-            return self.client.get(url="/v1/entries/", params=data)
-        finally:
-            self.client.close()
+        with httpx.Client(**self.settings) as client:
```

Comment:
Wouldn't it be nice to have some kind of helper method, that returns `Client` already instantiated with populated settings from one place?

```python
# In api/base.py

def get_client(self):
    return httpx.Client(**self.settings)


# In api/entries/v1.py and others

def list(...):
    with self.get_client() as client:
        return client.get(...)
```